### PR TITLE
feat: Adding a new page about seo.

### LINKED ISSUE #682: SEO page to accompany friendly link content
# Ensuring good SEO on Umbraco
Most of our traffic comes from search engines. It is important to set up SEO information for content pages.

If you do not use the SEO settings or review them regularly, your content will perform poorly.

## SEO Checker for content pages
We have installed an extension called SEO Checker on our content management system.

On every piece of content, you can set:
- [Focus keywords](https://yoast.com/focus-keyword/), keywords you want to rank the page for
- [An SEO title](https://yoast.com/page-titles-seo/), the title of the page for search engines
- [An SEO drescription](https://yoast.com/meta-descriptions/), the description of the page for search engines

You can also use the SEO Checker's validator to check for errors. Errors negatively impact your content's performance.

> Search engines change the way they rank search results all the time. You must review your content regularly otherwise your ranking might slip.

## SEO Checker for website issues
It is important that our website:
- Is kept up-to-date
- Does not have dummy content
- Has working links

Administrators have access to the SEO Checker tab on our content management system.

Use the **validate pages** tool to automatically check all content for SEO issues.

After you have run the tool under the **Issues** folder, you can see:
- Validation issues, errors with content
- Inbound link errors, links that users are trying to access but there is no content
- Configuration issues, errors with the set up of SEO checker

### Validation issues
The SEO checker will pick up content issues:
- Broken links, for example, a webpage on another website no longer exists
- Broken media
- Missing link titles
- Missing alternative text for images
- SEO title and description errors
- Lorem ipsum dummy text

You can fix the problem from the validation issues page. Click **edit** and the content page will appear in a pop up window.

### Inbound link errors
These are links where a user has tried to access a link but got an error page.

This can happen if someone has not set up a friendly link or you have a major content restructure.

You can choose where to redirect users who access that link. You can also see how many times the link has been accessed and where the last user accessed the link from.

> You do not need to redirect every inbound link. Sometimes they are typos or an old link.

### Configuration issues
You should talk to Digital and Data if SEO Checker picks up configuration issues.

### PR DESCRIPTION
**Pull request by catiarodriguescosta**
Fixes #682 
Preview link: https://deploy-preview-737--wmnetwork.netlify.app/styles/seo

### CODE REVIEW

File: src/www/pages/styles/seo/index.njk.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+## SEO Checker for content pages
+
+We have installed an extension called SEO Checker on our content management system.
+
+On every piece of content, you can set:
+- [Focus keywords](https://yoast.com/focus-keyword/), keywords you want to rank the page for
+- [An SEO title](https://yoast.com/page-titles-seo/), the title of the page for search engines
+- [An SEO drescription](https://yoast.com/meta-descriptions/), the description of the page for search engines
```
Comment (gldgrnt) [inline src/www/pages/styles/seo/index.njk.md:21]: Typo of 'description' here

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All good except for 1 typo! 👍🏾

**Review by gldgrnt — APPROVED** [review]
Nice nice nice

**Review by houbly — APPROVED** [review]

**Pull request merged**